src/Component/Histories/container.js

```diff
@@ -1,12 +1,18 @@
 import Histories from './Histories'
 import { connect } from 'react-redux'
-import { componentDidMount } from 'react-functional-lifecycle'
-import { getHistories, getNextHistories } from './../../Redux/State/History'
-import { compose } from 'ramda'
+import { componentDidMount, componentWillUnmount } from 'react-functional-lifecycle'
+import { clear, getHistories, getNextHistories } from './../../Redux/State/History'
+import { compose, groupBy } from 'ramda'
+
+// pageName :: String
+const pageName = 'history-page'
+
+// formatByDate :: Array -> Object
+const formatByDate = groupBy(history => new Date(history.createdAt).toLocaleDateString())
 
 // mapStateToProps :: State -> Props
 const mapStateToProps = state => ({
-  histories: state.History.histories,
+  histories: formatByDate(state.History.histories),
```

Comment:
This transformation should be done in the view to avoid continuous re-rendering (the fixes I made on i24news in the last PRs).